### DISCUSSION TITLE
TypeError: output tensor must have the same type as input tensor

### DISCUSSION THREAD

**Discussion by fatihayUtopia** [huggingface/autotrain-advanced#741] [Q&A]
after autotrain ended with nvidia 4xL4 ı got an error as i titled. ı got this error after autotrain ENDED. Everythink was okay in the training process.

**Comment by abhishekkrthakur** [comment]
which error?

**Reply by fatihayUtopia** [reply]
it said: TypeError: output tensor must have the same type as input tensor. and it did not save the model weights and other things
trained model: "fatihay/autotrain-1qjgk-oy0pj"

**Reply by fatihayUtopia** [reply]
by the way i trained "Qwen/Qwen2-1.5B-Instruct" base model and the training dataset was "Quardo/Turkish-Chat_GPT-4O" (chat column) and autotrain was on default settings i just set the chattemplate to chatml
